PSSwagger/Definitions.psm1

```diff
@@ -658,7 +658,11 @@ function New-SwaggerDefinitionCommand
         [Parameter(Mandatory=$false)]
         [AllowEmptyString()]
         [string]
-        $HeaderContent
+        $HeaderContent,
+
+        [Parameter(Mandatory=$false)]
+        [string]
+        $Formatter = 'None'
```

Comment:
Why not use parameter set like below?  [ValidateSet('None', 'PSScriptAnalyzer')]